src/main/java/uk/gov/hmcts/reform/cpo/controllers/RootController.java

```diff
@@ -6,9 +6,6 @@
 
 import static org.springframework.http.ResponseEntity.ok;
 
-/**
- * Default endpoints per application.
- */
 @RestController
 public class RootController {
```

Comment:
what is this RootController for?

Reply:
It was added by @smathangi - apparently:  `Azure application service has a hidden feature of making requests to root endpoint when "Always On" is turned on. This is the endpoint to deal with that and therefore silence the unnecessary 404s as a response code.`